.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_Direct_Java11.groovy

```diff
@@ -22,7 +22,7 @@ import PostcommitJobBuilder
 
 // This job runs the suite of ValidatesRunner tests using Java 11 against the Direct
 // runner compiled with Java 8.
-PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java11_ValidatesRunner_Direct',
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_Direct_Java11',
         'Run Direct ValidatesRunner in Java 11', 'Direct Runner ValidatesRunner Tests for Java 11', this) {
```

Comment:
Can we remove the `in` to make all names consistent in all invocations (same in the README) so it becomes `Run Direct ValidatesRunner Java 11`